tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,338 @@
+"""Tests for the Incremental Update feature"""
+
+from datetime import date, timedelta
+from fauxfactory import gen_alpha
+from nailgun import entity_mixins
+from nailgun.entities import (
```

Comment:
This is more a personal thing. I would use the same style as other parts in robottelo. This is also not a blocker.